server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1451,7 +1451,7 @@ public boolean performConsumerUpdates(ConsumerDTO updated, Consumer toUpdate,
 
             // this should update compliance on toUpdate, but not call the curator
             complianceRules.getStatus(toUpdate, null, false, false);
-            systemPurposeComplianceRules.getStatus(toUpdate, toUpdate.getEntitlements(), null, false, false);
+            systemPurposeComplianceRules.getStatus(toUpdate, toUpdate.getEntitlements(), null, true, true);
```

Comment:
We still want the updateConsumer parameter set to false here.

Reply:
Done.